### PR TITLE
Feature/propensity stratified evaluation tutorial

### PR DESCRIPTION
**Pull request by amirj**
### Description

I just added a new tutorial to represent propensity-based stratified evaluation.

### Checklist:

- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated `README.md` (if you are adding a new model).
- [ ] I have updated `examples/README.md` (if you are adding a new example).
- [ ] I have updated `datasets/README.md` (if you are adding a new dataset).

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -22,3 +22,8 @@ An important focus of Cornac is making it convenient to work with auxiliary data
 - [Working with auxiliary data](./working_with_auxiliary_data.md)
 - [Text to Graph transformation](./text_to_graph.ipynb)
 - [Visual recommendation algorithm with text data](./vbpr_text.ipynb)
+
+## Stratified Evaluation
```
Comment (tqtg) [inline tutorials/README.md:26]: Propensity Stratified Evaluation?
I think we should be consistent in the name of the method

**Comment by tqtg** [comment]
@saghiles please take a look and see if you have any comments on this PR?

File: tutorials/README.md (hunk 1 above)
Comment (amirj) [reply tutorials/README.md]: Good spot @tqtg --updated

**Comment by saghiles** [comment]
Hi @amirj, thanks for your contribution to Cornac, and congratulation for the good work on propensity stratified evaluation. I went through the tutorial and my overall feedback is as follows.

The current version reads more like a usage example of the ``PropensityStratifiedEvaluation`` evaluation method. One would expect the tutorial to be self-contained, it should be possible to understand the core idea behind the proposed propensity-based stratified evaluation without (or very little) reference to the original paper. We can think of it as a blog post describing your contribution along with some code illustrating how to use this new evaluation method in Cornac. 

If you think that this may require significant efforts and may take some time to do, I would recommend leveraging the material (code) of the current tutorial to make an example first. Please see Cornac [examples](https://github.com/PreferredAI/cornac/tree/master/examples) format, the example should be a runnable python file supported with comments. If you agree please feel free to create a PR accordingly. 

We can then integrate a tutorial on  propensity-based stratified evaluation once we have enough materials. 

Please let us know what are your thoughts, and thanks again for your contribution.

**Comment by tqtg** [comment]
@amirj do you want to convert this into an example first? Maybe we can prepare a more comprehensive tutorial in the form of notebook with more materials and guiding explanation for users?

**Comment by amirj** [comment]
Thanks @saghiles and @tqtg and sorry for my late response.
Yes, let's convert this into an example first. I'll check the format of examples to convert it. In the meantime, let me know if you have further comments. Thanks.

**Comment by amirj** [comment]
@saghiles @tqtg I just modified this PR to reflect a usage example of propensity stratified evaluation method.

**Comment by tqtg** [comment]
I just did some minor changes and updated the docs. Overall, I'm good with the example.

@saghiles any further comments?

**Comment by saghiles** [comment]
The example looks good to me. If @amirj  has no further changes we can merge this PR.

**Comment by amirj** [comment]
Thanks both. It looks great to me. Nice to see people can easily reproduce propensity-based stratified evaluation for any model available in Cornac framework :-)

**Comment by saghiles** [comment]
Since there are no more updates, I'll go ahead and merge this PR. Thanks all!

**Pull request merged**